src/app/dim-api/selectors.ts

```diff
@@ -22,4 +22,4 @@ export const currentProfileSelector = createSelector(
  * TODO: Sort/trim this list
  */
 export const recentSearchesSelector = (state: RootState) =>
-  state.dimApi.searches[destinyVersionSelector(state)];
+  state.dimApi.searches?.[destinyVersionSelector(state)];
```

Comment:
https://sentry.io/share/issue/a18b92493cde444db4eeb77af7dab091/

Reply:
This shouldn't be necessary - I think I fixed it higher up by making sure that searches is always defined.